frontend/next.config.js

```diff
@@ -0,0 +1,70 @@
+// for pages that are not in cclw's sitemap
+const cclwRedirects = [
+    { source: "/cookie-policy", destination: "/", permanent: true },
+    { source: "/privacy", destination: "/", permanent: true },
+];
+
+// for pages that are not in cpr's sitemap
+const cprRedirects = [
+    { source: "/about", destination: "/", permanent: true },
+    { source: "/acknowledgements", destination: "/", permanent: true },
+    { source: "/contact", destination: "/", permanent: true },
+    { source: "/methodology", destination: "/", permanent: true },
+];
+
+const defaultRedirects = [
+  {
+      source: "/auth/:id*",
+      destination: "/",
+      permanent: true,
+    },
+    {
+      source: "/account",
+      destination: "/",
+      permanent: true,
+    },
+    {
+      source: "/users/:id*",
+      destination: "/",
+      permanent: true,
+    },
+    {
+      source: "/litigation/:id*",
+      destination: "/",
+      permanent: false, // will become a page eventually
+    },
+  ];
+ 
+
+function get_redirects() {
+    const REDIRECT_FILE = process.env.NEXT_REDIRECT_FILE || "default.json";
```

Comment:
I think this will only ever return the default.json because from my understanding the config is only executed at build time